okhttp/src/main/kotlin/okhttp3/internal/platform/ConscryptPlatform.kt

```diff
@@ -83,9 +83,7 @@ class ConscryptPlatform private constructor() : Platform() {
   override fun newSslSocketFactory(trustManager: X509TrustManager): SSLSocketFactory {
     return newSSLContext().apply {
       init(null, arrayOf<TrustManager>(trustManager), null)
-    }.socketFactory.also {
-      Conscrypt.setUseEngineSocket(it, true)
```

Comment:
no longer necessary?

Reply:
It's the default as per the release notes